WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -639,4 +630,19 @@ public void onPostUploaded(OnPostUploaded event) {
             loadPosts(LoadMode.FORCED);
         }
     }
+
+    /*
+     * Media info for a post's featured image has been downloaded, tell
+     * the adapter so it can show the featured image now that we have its URL
+     */
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onMediaChanged(MediaStore.OnMediaChanged event) {
+        if (isAdded() && !event.isError() && getPostListAdapter() != null) {
```

Comment:
Calling `getPostListAdapter()` will create the adapter if it doesn't already exist, so instead we should use `mPostsListAdapter != null` here.